src/connection/ClientConnection.cpp

```diff
@@ -254,7 +274,7 @@ void ClientConnection::setupClientConnection(std::list<ServerStruct> *serverStru
 			pollfds.push_back(connection.second.pfd);
 		}
 
-		int poll_count = poll(pollfds.data(), pollfds.size(), 10);
+		int poll_count = poll(pollfds.data(), pollfds.size(), TIMEOUT * 1000);
```

Comment:
niet zeker van of deze timeout hetzelfde is

Reply:
https://man7.org/linux/man-pages/man2/poll.2.html
"The timeout argument specifies the number of milliseconds that
       poll() should block waiting for a file descriptor to become
       ready.  The call will block until either: •  a file descriptor becomes ready;

       •  the call is interrupted by a signal handler; or

       •  the timeout expires.
"
Inderdaad, kennelijk blokt het dan voor x milliseconden, tenzij er een fd ready is of er een interrupt is.
Al maakt het niet echt uit als er geblokt wordt als er geen fd ready is? Ik kan hier niet echt goede recommendations voor vinden..